installpresto.sh

```diff
@@ -32,11 +32,19 @@ if [[ $(hostname -s) = hn* ]]; then
   wget https://repo1.maven.org/maven2/com/facebook/presto/presto-cli/$VERSION/presto-cli-$VERSION-executable.jar -O /usr/local/bin/presto-cli
   chmod +x /usr/local/bin/presto-cli
 
-  until slider registry  --name presto1 --getexp presto ; do
+  retry=0
+
+  until [[ slider registry  --name presto1 --getexp presto || retry -ge 60 ]]; do
     echo "waiting for presto to start.."
```

Comment:
May be make message more useful, like 
"waiting for presto to start.. attempt $retry/60"